quartz/components/scripts/search.inline.ts

```diff
@@ -248,6 +288,42 @@ document.addEventListener("nav", async (e: unknown) => {
     }
   }
 
+  async function fetchContent(slug: FullSlug): Promise<Element[] | undefined> {
+    if (fetchContentCache[slug]) {
```

Comment:
We can do better here if we save this cache to localStorage. @jackyzha0 what's your opinion here?

Reply:
Seems a bit overkill imo, would also require you to handle localStorage not getting too full and handling cache invalidation, but let's see what Jacky says

Reply:
imo in memory cache is fine, let's not persist to localStorage